about.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Annual Diabetes Walk About Page</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <nav class="social">
+      <ul>
+        <li>
+          <a href=""><i class="fa-brands fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+        </li>
+        <li><a href=""></a></li>
+        <li><a href="">English</a></li>
+        <li><a href="">My Page</a></li>
+        <li><a href="">Logout</a></li>
+      </ul>
+    </nav>
+    <div id="mobile-overlay">
+      <i class="fa-solid fa-xmark"></i>
+      <nav class="nav">
+        <div class="official-logo">
+          <img
+            class="official-logo-img"
+            height="160"
+            width="200"
+            src="images/annual-diabetes-walk-logo.png"
+            alt=""
+          />
+        </div>
+        <ul>
+          <li class="mobile-nav-link"><a href="index.html">Home</a></li>
+          <li class="mobile-nav-link"><a href="index.html">Program</a></li>
+          <li class="mobile-nav-link"><a href="index.html">Join</a></li>
+          <li class="mobile-nav-link"><a href="index.html">Sponsor</a></li>
+          <li class="mobile-nav-link"><a href="index.html">News</a></li>
+          <li class="mobile-nav-link"><a href="about.html">About</a></li>
+          <li class="mobile-nav-link"><a href="index.html">CC Campaign</a></li>
+        </ul>
+      </nav>
+    </div>
+    <section id="header" class="about">
+      <header class="about-header">
+        <i class="fa-solid fa-bars"></i>
+        <h1>Annual Diabetes Walk</h1>
+        <h1>in nairobi 2022</h1>
```

Comment:
- [ ] Kindly implement the change requested by the previous reviewer. He suggested you not to use `h1 `tag more than once per page because it is a bad practice. Instead you do like that as he suggested 👇 
- `<h1>Annual Diabetes Walk <br /> IN NAIROBI 2022</h1>`